src/attribute.h

```diff
@@ -30,8 +35,43 @@ class Integer: public Attribute
 {
     public:
         Integer(uint8_t type, const uint8_t* data, size_t size);
+        Integer(uint8_t type, uint32_t value);
         std::string value() const override;
+        std::vector<uint8_t> toVector(std::string secret, std::array<uint8_t, 16> auth) const override;
     private:
         uint32_t m_value;
 };
+
+class IpAddress : public Attribute
+{
+    public:
+        IpAddress(uint8_t type, const uint8_t* data, size_t size);
+        IpAddress(uint8_t type, std::string address);
+        std::string value() const override;
+        std::vector<uint8_t> toVector(std::string secret, std::array<uint8_t, 16> auth) const override;
+    private:
+        std::string m_value;
```

Comment:
IP address inner representation must be either `uint32_t` or `std::array<uint8_t, 4>`.